<?xml version="1.0"?>
<launch>
    <arg name= "platformID" default="right"/>
    
    <group ns="$(arg platformID)">
        <node pkg="foot_platform_ros" type="footForceMeasModifier" name="force_sensor_modifier" output="screen">
            <param name="platformID" value="$(arg platformID)"/>
            <remap from="robot_description" to="platform_description" />
        </node>
    </group>

</launch>